tests/cli.rs

```diff
@@ -530,6 +532,28 @@ fn user_password_auth() {
     mock.assert();
 }
 
+#[test]
+fn netrc_user_password_auth() {
+    let server = MockServer::start();
+    let mock = server.mock(|when, _then| {
+        when.header("Authorization", "Basic dXNlcjpwYXNz");
+    });
+
+    let mut netrc = tempfile::NamedTempFile::new().unwrap();
+    writeln!(
+        netrc,
+        "machine {}\nlogin user\npassword pass",
+        server.host()
+    )
+    .unwrap();
+
+    get_command()
+        .env("NETRC", netrc.path().to_str().unwrap())
```

Comment:
`.env()` takes `AsRef<OsStr>`, so I think this would just work:
```suggestion
        .env("NETRC", netrc.path())
```